.github/workflows/issue-label.yml

```diff
@@ -0,0 +1,17 @@
+on:
+  issues:
+    types: [labeled]
+
+env:
+  DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
+
+jobs:
+  labelNotify:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: notify
+        if: github.event.label.name == 'enhancement' || github.event.label.name == 'bug'
```

Comment:
Oh...I think the example I had found was looking at `event.issue.labels`, but instead it looks like what you have used is exactly what I was wanting in the first place in pr.yml.